iotdb-core/datanode/src/main/java/org/apache/iotdb/db/protocol/rest/v2/impl/RestApiServiceImpl.java

```diff
@@ -265,12 +311,21 @@ public Response insertTablet(
               partitionFetcher,
               schemaFetcher,
               config.getQueryTimeoutThreshold());
-
+      finish = true;
       return responseGenerateHelper(result);
     } catch (Exception e) {
       return Response.ok().entity(ExceptionHandler.tryCatchException(e)).build();
     } finally {
+      long costTime = System.nanoTime() - startTime;
+      if (insertTabletStatement != null) {
+        addStatementExecutionLatency(
+            OperationType.INSERT_TABLET, insertTabletStatement.getType().name(), costTime);
+      }
       if (queryId != null) {
+        long executionTime = COORDINATOR.getTotalExecutionTime(queryId);
+        if (finish)
+          addQueryLatency(
+              insertTabletStatement.getType(), executionTime > 0 ? executionTime : costTime);
```

Comment:
I think the insert interfaces don't need to record the `Query` latency.

Reply:
well,  I'll remove it